AppDB/cassandra_env/prime_cassandra.py

```diff
@@ -1,104 +1,114 @@
-    print "Received an exception of type " + str(e.__class__) +\
-          " with message: " + str(e)
-
-  print "CASSANDRA SETUP SUCCESSFUL"
-  return True
+  uaserver_tables = [
+    {'name': dbconstants.APPS_TABLE, 'schema': dbconstants.APPS_SCHEMA},
+    {'name': dbconstants.USERS_TABLE, 'schema': dbconstants.USERS_SCHEMA}
+  ]
+  for table in uaserver_tables:
+    key = bytearray('/'.join([dbconstants.SCHEMA_TABLE, table['name']]))
+    columns = bytearray(':'.join(table['schema']))
+    define_schema = """
+        INSERT INTO "{table}" ({key}, {column}, {value})
+        VALUES (%(key)s, %(column)s, %(value)s)
+      """.format(table=dbconstants.SCHEMA_TABLE,
+                 key=ThriftColumn.KEY,
+                 column=ThriftColumn.COLUMN_NAME,
+                 value=ThriftColumn.VALUE)
+    values = {'key': key,
+              'column': dbconstants.SCHEMA_TABLE_SCHEMA[0],
+              'value': columns}
+    session.execute(define_schema, values)
+
 
 def prime_cassandra(replication):
-  """
-  Create required tables for AppScale
+  """ Create Cassandra keyspace and initial tables.
 
   Args:
-    replication: Replication factor of data
+    replication: An integer specifying the replication factor for the keyspace.
   Raises:
-    AppScaleBadArg if replication factor is not greater than 0
-    Cassandra specific exceptions upon failure
-  Returns:
-    0 on success, 1 on failure. Passed up as process exit value.
-  """ 
-  if int(replication) <= 0: 
-    raise dbconstants.AppScaleBadArg("Replication must be greater than zero")
-
-  create_keyspaces(int(replication))
-  print "Prime Cassandra database"
-  db = py_cassandra.DatastoreProxy()
-  try:
-    db.create_table(dbconstants.USERS_TABLE, dbconstants.USERS_SCHEMA)
-    db.create_table(dbconstants.APPS_TABLE, dbconstants.APPS_SCHEMA)
-  # TODO: Figure out the exact exceptions we're trying to catch in the 
-  # case where we are doing data persistance
-  except Exception, e:
-    print "Received an exception of type " + str(e.__class__) +\
-          " with message: " + str(e)
-
-  if len(db.get_schema(dbconstants.USERS_TABLE)) > 1 and \
-     len(db.get_schema(dbconstants.APPS_TABLE)) > 1:
-    print "CREATE TABLE SUCCESS FOR USER AND APPS"
-    print "USERS:",db.get_schema(dbconstants.USERS_TABLE)
-    print "APPS:",db.get_schema(dbconstants.APPS_TABLE)
-    return 0
-  else: 
-    print str(db.get_schema(dbconstants.USERS_TABLE))
-    print str(db.get_schema(dbconstants.APPS_TABLE))
-    print "FAILED TO CREATE TABLE FOR USER AND APPS"
-    return 1
+    AppScaleBadArg if replication factor is not greater than 0.
+    TypeError if replication is not an integer.
+  """
+  if not isinstance(replication, int):
+    raise TypeError('replication must be an integer')
+
+  if int(replication) <= 0:
+    raise dbconstants.AppScaleBadArg('Replication must be greater than zero')
+
+  with open(MASTERS_FILE_LOC) as masters_file:
```

Comment:
Is ti possible to use the functions in lib/appscale_info.py instead of doing the same work here for all db nodes?